### ISSUE TITLE
Bambu Studio prevents the display turning off / computer sleeping if you have watch the printer camera video and then stopped it but stayed on the device tab

### ISSUE THREAD

**Issue by AdrianGarside** [bambulab/BambuStudio#2664]
### Bambu Studio Version

1.7.0

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 11

### Printer

P1P

### How to reproduce

1. View camera feed from printer
2. Stop the feed
3. Run 'powercfg /requests' in an elevated command prompt

BUG:
Display and System are blocked by bambu studio. The display will never turn off and the machine will never sleep.

4. Navigate away from the device tab and the blocked state will resolve.

### Actual results

Display stays on. Machine never sleeps.

### Expected results

Having stopped the video there is no reason Bambu Studio to force the display to stay on or to block the machine from sleeping.

### Project file & Debug log uploads

N/A

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by bambu123** [comment]
There is a feature request relate to this.

**Comment by AdrianGarside** [comment]
Do you know which release will have the fix?

**Comment by AdrianGarside** [comment]
Oops, jinxed it. This time when I stopped the video playback, I hit the bug:
![image](https://github.com/bambulab/BambuStudio/assets/26147576/738ea4d0-16ff-466c-8b6c-67c54657ba2c)
Now nothing I do corrects it other than closing bambu studio.

**Comment by bambu123** [comment]
We haven't plan to work on the feature. But this is a duplicate issue.
You can post your request here. #2477

**Comment by AdrianGarside** [comment]
@bambu123 that issue is complaining about a technically by design behavior - video play back on windows generally should stop the display (and machine) turning off. 

This is complaining about bugs in the current behavior where Bambu Studio often blocks the display turning off or the machine going to sleep permanently having ever viewed (and then stopped) the video preview.

Are you going to be blocking the current behavior completely under #2477?

**Comment by ArnieO** [comment]
I have this issue on a new installation. Hoping now that this fix remedies it: https://github.com/bambulab/BambuStudio/issues/2477#issuecomment-1716304349

**Comment by PunaJussi** [comment]
This really should be fixed.

**Comment by bryanhunwardsen** [comment]
I can confirm this is fixed on current release as of today (at least with a single instance of bambu studio open and connected to a single/active machine. I checked the power state lock before , during and after preview is stopped and bamabu studio cleaned the lock up after video stream stopped.

There may be some edge cases (I have logged in one of the linked issues) that orphan the lock but I can verify for Windows 10, the simple case of removing the lock when stream is stopped is working.
